app/helpers/sidebar_helper.rb

```diff
@@ -7,6 +7,7 @@ def active_class(link_path)
     controller_name = link_path.split("/").second
     is_active = current_page?(controller: controller_name, action: :index) ||
```

Comment:
BTW, I'm not particularly happy with this solution so if you can think of a "more ideal" implementation, let me know.

Reply:
It appears that omitting the `action` argument does not cover all cases we need.